video/out/vo_gpu_next.c

```diff
@@ -912,6 +912,29 @@ static void apply_crop(struct pl_frame *frame, struct mp_rect crop,
     }
 }
 
+static void update_tm_viz(struct pl_color_map_params *params,
+                          const struct pl_frame *target,
+                          double pts)
+{
+    if (!params->visualize_lut)
+        return;
+
+    // Use right half of sceen for TM visualization, constrain to 1:1 AR
+    const float out_w = fabsf(pl_rect_w(target->crop));
+    const float out_h = fabsf(pl_rect_h(target->crop));
+    const float size = MPMIN(out_w / 2.0f, out_h);
+    params->visualize_rect = (pl_rect2df) {
+        .x0 = 1.0f - size / out_w,
+        .x1 = 1.0f,
+        .y0 = 0.0f,
+        .y1 = size / out_h,
+    };
+
+    // Complete one full rotation of the hue plane every 10 seconds
+    const float tm_period = 10.0;
+    params->visualize_hue = 2 * M_PI * pts / tm_period;
```

Comment:
Would be fun if it could rotate with one frame or in pause, but we don't render in this case.

Reply:
The commit message explains why I didn't go with this approach. I think the current compromise is good enough. This is just meant to look fancy, probably most users will not understand the graph anyway. /s

Reply:
> This is just meant to look fancy, probably most users will not understand the graph anyway. /s

:)